autoload/neomake/makers/mlint.vim

```diff
@@ -0,0 +1,12 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#maker#mlint#mlint() abort 
```

Comment:
> Error: Vim(function):E746: Function name does not match script file name: neomake#maker#mlint#mlint (in /testplugin/autoload/neomake/makers/mlint.vim

You should move it to `./autoload/neomake/makers/ft/mlint.vim` and name it `neomake#makers#ft#mlint#mlint`.